File: #1-25 (C, Java, Python)/Python/Euler6.py
#Euler6 ~ Sum square difference
import timeit
start = timeit.default_timer()

sumx = 0
for i in range(1,101):
    sumx = sumx + pow(i, 2)
    
sumy = 0
for j in range(1,101):
    sumy = sumy + j    
sumy = pow(sumy,2)
diff = sumy - sumx


stop = timeit.default_timer()
print("Solution: ",diff,"\n")
print("Program completed in","{0:.3f}".format(stop - start), "seconds.")

File: #1-25 (C, Java, Python)/C/Euler6.c
//Euler6 ~ Sum square difference
#include <time.h>
#include <stdio.h>
#include <math.h>

clock_t start, end;
double cpu_time_used;

int main()
{
	start = clock();
	
	double sumx = 0;
		for(int i = 1; i <= 100; i++)
		{
			sumx = sumx + pow(i, 2);
		}
		
		double sumy = 0;
		for(int j = 0; j <= 100; j++)
		{
			sumy = sumy + j;
		}
		sumy = pow(sumy, 2);
		
		double diff = sumy - sumx;
	
	end = clock();
	cpu_time_used = ((double) (end - start)) / CLOCKS_PER_SEC;
	printf("Solution: %f\n\n",diff);
	printf("Program completed in %.3f seconds.", cpu_time_used);
}